src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const SEX_MALE = 'm';
+  const isWifeOrHusband = sex === SEX_MALE
+    ? `${partnerName} is my wife`
+    : `${partnerName} is my husband`;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && <p className="Person__age">{age && `I am ${age}`}</p>}
```

Comment:
This code is not readable, let's move the paragraph to the new line
```suggestion
      {age && (
        <p className="Person__age">{age && `I am ${age}`}</p>
      )}
```

Reply:
Also it's not clear why did you check `age` variable 2 times here
Inner check is redundant, delete it